engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -4128,6 +4129,7 @@ public void mainHelper() throws Exception
 
   public static void main(final String[] args) throws Exception
   {
+    LoggerUtil.addAppender("apexclient");
```

Comment:
My issue is using the application attribute for cli. dt.attr.LOGGER_APPENDER is a short form for dt.application.*.attr.LOGGER_APPENDER and is still an application attribute. The cli shouldn't have a dependency to this attribute, even if it is just for naming such as attribute.getLongName(). It should be a separate cli property.